raiden/ui/console.py

```diff
@@ -216,27 +224,6 @@ def register_token(self, token_address_hex):
         self._raiden.register_channel_manager(channel_manager.address)
         return channel_manager
 
-    def ping(self, peer_address_hex, timeout=4):
```

Comment:
Because we use the ping to do healthcheck and the value of the `nonce` is important, the user should not mess up with the health check task state.